openedx/adg/lms/webinars/models.py

```diff
@@ -638,7 +649,48 @@ class WebinarRegistration(TimeStampedModel):
 
     class Meta:
         app_label = 'webinars'
-        unique_together = ('webinar', 'user')
+        unique_together = ('webinar', 'email')
 
     def __str__(self):
-        return f'User {self.user_id}, Webinar {self.webinar_id}'
+        return f'Email: {self.email} , Webinar: {self.webinar_id}'
+
+    @classmethod
+    def create_team_registrations_and_schedule_reminder(cls, team_member_emails, webinar, **kwargs):
+        """
+        Create or update webinar team registrations and schedule reminder emails for
+        the given members for this webinar
+
+        Args:
+            team_member_emails (iterable): List of emails for which team registrations will be added.
+            webinar (Webinar): Webinar for which registrations will be added.
+            **kwargs (dict): Dictionary of values to be set for the registrations.
+        """
+        for team_member_email in team_member_emails:
+            WebinarRegistration.objects.update_or_create(
+                webinar=webinar,
+                email=team_member_email,
+                defaults={
+                    'is_team_member_registration': True,
+                    **kwargs
+                }
+            )
+        schedule_webinar_reminders(list(team_member_emails), webinar)
```

Comment:
A webinar is published, A user(not a team member) registers in it so reminders are scheduled. When the webinar is updated then the same user is added as panelist. In that case we will again schedule reminders for that user so he will receive duplicate emails. Right?